backend/app/infrastructure/database/score_delegation.py

```diff
@@ -0,0 +1,17 @@
+from app.infrastructure.database.models import ScoreDelegation
+
+
+def save_delegation(
+    primary_addr_hash: str, secondary_addr_hash: str, both_addr_hash: str
+):
+    delegation_primary = ScoreDelegation(hashed_addr=primary_addr_hash)
+    delegation_secondary = ScoreDelegation(hashed_addr=secondary_addr_hash)
+    delegation_both = ScoreDelegation(hashed_addr=both_addr_hash)
+
+    delegation_primary.add()
```

Comment:
have you tested if calling .add() on model instead of db.session.add() is a valid solution? 

+ i see we haven't written tests for this service - maybe we can cosider adding them?

Reply:
and migration file is missing, i guess

Reply:
I haven't tested it yet, because I'm waiting for https://github.com/golemfoundation/octant/pull/221 to be merged. So the service test and migration will be added after that

Reply:
sure, so lgtm

Reply:
reverted to db.session.add(), because it didn't work :) Tests added, migration added